cpp/include/cuvs/preprocessing/quantize/binary.hpp

```diff
@@ -59,17 +76,20 @@ void transform(raft::resources const& res,
  * Usage example:
  * @code{.cpp}
  * raft::handle_t handle;
+ * cuvs::preprocessing::quantize::binary::params params;
  * auto quantized_dataset = raft::make_host_matrix<uint8_t, int64_t>(handle, samples,
- * features); cuvs::preprocessing::quantize::binary::transform(handle, dataset,
+ * features); cuvs::preprocessing::quantize::binary::transform(handle, params, dataset,
```

Comment:
Please consider the following potential use case:
```c++
cuvs::preprocessing::quantize::binary::transform(handle, params, dataset, quantized_dataset.view());
auto index = cagra::build(handle, build_params, quantized_dataset);

auto quantized_queries = raft::make_device_matrix<uint8_t, int64_t>(handle, queries.extent(0), queries.extent(1));
cuvs::preprocessing::quantize::binary::transform(handle, params, queries, quantized_queries.view());

cagra::search(handle, search_params, index, quantized_queries, neighbors, distances);
```

How do we quantize the `queries` without storing the quantization parameters?